setup.py

```diff
@@ -1,24 +0,0 @@
-from setuptools import setup
```

Comment:
It might be worth keeping a shim version of this file. I'm not sure. 

```python
import setuptools

if __name__ == "__main__":
    setuptools.setup()
```

https://stackoverflow.com/a/62983901

Reply:
>If you need a `setup.py` for an editable install

You don't need it thanks to my PR into pip ;) You onpy need the latest `pip`.